Maths/FindMedian.java

```diff
@@ -0,0 +1,134 @@
+ackage ads.set02.median;
```

Comment:
use `package ads.set02.median` 👍🏻

Reply:
Oh, good point. That still is the wrong package. Thanks

Reply:
Thanks. Still is the wrong package.